src/dungeon_ai_item_weight.c

```diff
@@ -219,7 +219,7 @@ u32 GetAIUseItemProbability(Entity *targetPokemon, Item *item, u32 itemTargetFla
             }
             break;
         case ITEM_CHERI_BERRY:
-            if (pokemonInfo->nonVolatile.nonVolatileStatus != STATUS_PARALYSIS)
+            if (pokemonInfo->burnClassStatus.status != STATUS_PARALYSIS)
```

Comment:
So burn class has both paralysis and poison? This name doesn't make a lot of sense.. where does this come from in sky? Debug strings?

Reply:
The Sky names are from pmdsky-debug (i.e., documented by their hacking community), and are based on the status with an ID of 1 for the respective status slot. I never liked the names I gave to these status variables as a lot of them are inaccurate or confusing. For example, "non-volatile" status" in the main games includes both sleep and freeze as well, which are not grouped together with burn/paralysis/poison in Mystery Dungeon. IMO the statuses are so varied that it's hard to come up with good names for them, so I prefer the Sky community's more orderly approach.

Reply:
Gotcha. Still don't like it but least there's a little explanation for it.